src/main/java/core/DataProviders.java

```diff
@@ -0,0 +1,30 @@
+package core;
+
+import beans.Board;
+
+import constants.ParameterName;
+import org.testng.annotations.DataProvider;
+
+public class DataProviders {
+
+    @DataProvider
+    public static Object[][] trelloBoardDataProvider() {
+        return new Object[][]{
+            {Board.generateString(10)}
+        };
+    }
+
+    @DataProvider
+    public static Object[][] trelloBoardWithUpdatesDataProvider() {
+        return new Object[][]{
+            {Board.generateString(10), "NewCardName"}
+        };
+    }
+
+    @DataProvider
+    public static Object[][] getFieldOnBoard() {
+        return new Object[][]{
+            {Board.generateString(10), ParameterName.ID_MEMBER_CREATOR}
```

Comment:
if you use generateString(int num) method only in this class, why not create it in this class?

Reply:
generateString(int) is declared in this class now